src/components/contentLibrary/contentLibrary.component.tsx

```diff
@@ -40,54 +78,124 @@ export const ContentLibrary = () => {
     [objectListingWidth, windowSize],
   );
 
+  const sensors = useSensors(
+    useSensor(PointerSensor, {
+      activationConstraint: {
+        delay: 100,
+        tolerance: 5,
+      },
+    }),
+  );
+
+  const primaryKey = draggedObject && getPrimaryKey(draggedObject);
+
   return (
-    <div
-      className="flex h-screen flex-row"
-      style={{
-        maxHeight: `calc(100vh - 5rem)`,
+    <DndContext
+      onDragStart={handleDragStart}
+      onDragEnd={handleDragEnd}
+      sensors={sensors}
+      measuring={{
+        draggable: {
+          measure: (element) => {
```

Comment:
if this is a function that is called loads when you drag, maybe just get the mouse position in this?